pkg/kube/wait.go

```diff
@@ -39,9 +39,10 @@ import (
 )
 
 type waiter struct {
-	c       ReadyChecker
-	timeout time.Duration
-	log     func(string, ...interface{})
+	c       						ReadyChecker
+	timeout 						time.Duration
+	perResourceTimeout 	time.Duration
+	log     						func(string, ...interface{})
```

Comment:
Formatting is broken, see [Circle CI](https://app.circleci.com/pipelines/github/helm/helm/4552/workflows/d62a68a0-f08e-472c-9ce4-42887a54b268/jobs/16127?invite=true#step-103-7)

Let me know, if you need help with `go fmt`